### DISCUSSION TITLE
doi's used in bonares typically do not resolve in crossref or openaire

### DISCUSSION THREAD

**Discussion by pvgenuchten** [soilwise-he/harvesters#85] [Announcements]
What is the typical mechanism to make intermediaries such as OpenAire and Crossref aware of DOI's?

Asking because some of the Bonares DOI's seems not available in OpenAire and Crossref, others are

See https://api.crossref.org/swagger-ui/index.html#/Works/get_works__doi_ and try any of the doi's advertised in https://tools.bonares.de/doi/datasets/

Same for https://explore.openaire.eu/search/advanced/research-outcomes?f0=pid&fv0=10.20387%252Fbonares-1yhh-07w1, but findable for https://explore.openaire.eu/search/advanced/research-outcomes?f0=pid&fv0=10.20387%252Fbonares-43xt-vjpb

On the other hand [Datacite](https://commons.datacite.org/?query=10.20387%2Fbonares-1yhh-07w1) and DOI itself returns them perfectly:

send a GET request to https://doi.org/10.20387/bonares-1yhh-07w1 with accept: application/x-bibtex

```
@misc{https://doi.org/10.20387/bonares-1yhh-07w1,
  doi = {10.20387/BONARES-1YHH-07W1},
  url = {https://maps.bonares.de/mapapps/resources/apps/bonares/index.html?lang=en&mid=b0b98713-a12e-47df-8911-c66305fd0c02},
  author = {Phalempin, Maxime},
  title = {Loam/Sand fitted parameters},
  publisher = {ZALF(Leibniz Centre for Agricultural Landscape Research (ZALF))},
  year = {2024},
  copyright = {Creative Commons Attribution 4.0 International}
}
```

For ISRIC DOI's such as 10.17027/isric-soilgrids.c6cb5073-78dd-4d8d-be81-9d546a1c004f, 10.17027/isric-soilgrids.f0797a68-1692-11ea-a7c0-a0481ca9e724, 10.17027/isric-soilgrids.713396fc-1687-11ea-a7c0-a0481ca9e724 , these are actually findable in OpenAire, but not in Crossref

[Mendeley](https://data.mendeley.com) is one of the platforms which was actually able to resolve some of the [Bonares](https://data.mendeley.com/research-data/?repositoryType=NON_ARTICLE_BASED_REPOSITORY&search=(10.20387)) and [isric](https://data.mendeley.com/research-data/?repositoryType=NON_ARTICLE_BASED_REPOSITORY&source=delft.isric&search=(10.17027)) data sources

**Comment by cdonmez16** [comment]
Thanks for bringing this up. This is currently true that some of the Bonares DOI's are not available in OpenAire and Crossref. Becaues, at this stage, we use DataCite for data, not CrossRef which is typically used for publications. We aim to join OpenAIRE if we achieve certification from CTS, which is close (likely in 2025). Additionally, we are already part of OpenDOAR. - https://v2.sherpa.ac.uk/opendoar/
@pvgenuchten
